clients/banking/src/components/TransferInternationalWizardBeneficiary.tsx

```diff
@@ -147,16 +160,37 @@ export const TransferInternationalWizardBeneficiary = ({
     listenFields(["results"], () => refresh());
   }, [listenFields, refresh]);
 
+  useEffect(() => {
+    if (isNotNullish(errors) && errors.length) {
+      for (const error of errors) {
+        if (isNotNullishOrEmpty(error[0])) {
+          dynamicFormApiRef.current?.setDynamicFieldError(error[0], error[1]);
+        }
+      }
+    }
+  }, [setFieldError, errors, dynamicFormApiRef.current?.setDynamicFieldError]);
+
+  const orphanErrors = useMemo(() => errors?.filter(([key]) => isNullish(key)), [errors])?.map(
+    ([_, message]) => message,
+  );
```

Comment:
Could be removed entirely as there's no need to check if first tuple item is nullish.